fanout.go

```diff
@@ -149,3 +150,20 @@ func (f *Fanout) isAllowedDomain(name string) bool {
 	}
 	return true
 }
+
+func (f *Fanout) processClient(ctx context.Context, c Client, r *request.Request) *response {
+	start := time.Now()
+	for j := 0; j < f.attempts || f.attempts == 0; {
+		if ctx.Err() != nil {
+			return &response{client: c, response: nil, start: start, err: ctx.Err()}
+		}
+		msg, err := c.Request(r)
```

Comment:
Should it be delay between attempts?

Reply:
yes, I think it makes sense. Thanks, fixed.